app/domain/conjur/fetch_required_secrets.rb

```diff
@@ -0,0 +1,53 @@
+require 'command_class'
+require 'util/error_class'
+
+module Conjur
+
+  # TODO: really, these errors should be inside FetchRequiredSecrets but that
```

Comment:
If this is technical debt that should be fixed, would you mind adding an issue for it?